libs/build_gftl_shared.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eux
+
+name="gftl-shared"
+repo=${1:-${STACK_gftl_shared_repo:-"Goddard-Fortran-Ecosystem"}}
```

Comment:
Same comment as above for esma-cmake and CMakeModules.

Reply:
Done in 81e539f4a6443c4253b9e4825eeeebcd28bda884